modules/microshift-creating-ostree-iso.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-creating-ostree-iso_{context}"]
+= Creating the {op-system-ostree-first} image
+
+Use the following procedure to create the ISO. The {op-system-ostree} Installer image pulls the commit from the running container and creates an installable boot ISO with a Kickstart file configured to use the embedded OSTree commit.
+
+.Prerequisites
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the `composer-cli` tool.
+* You have root-user access to your build host.
+* You have the `podman` tool.
+//Q: Should podman have come with a RHEL package? Is it just `# yum install podman`? We can make that step #1 if easier for our users.
```

Comment:
It's just `sudo dnf install -y podman`, however I would leave it as-is. 
If we start giving commands, we should also give those for other prereqs

Reply:
Interesting, the RHEL docs DO give single commands in the prereqs, for example here: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/building_running_and_managing_containers/using-the-container-tools-api 
I'm not sure what our rules on this are in OCP,  but I can check. 
I cannot find a good "additional resource" link for users that aren't familiar with podman.